### DISCUSSION TITLE
How to build bundle like operator-sdk?

### DISCUSSION THREAD

**Discussion by tuhao1020** [kubernetes-sigs/kubebuilder#4164] [Support]
How to build bundle like operator-sdk?

**Comment by camilamacedo86** [comment]
> How to build bundle like operator-sdk?
 
 Do you want to buiild a bundle to publish your solution in OCP/OperatorHub io? Or do you want to only distribute your solution for your users ?
 
 What exactly do you mean?
 Can you please bring more context about what you want to achieve so that we might help you in a better way

**Comment by camilamacedo86** [comment]
@tuhao1020 

We are closing this one since no enough info was provided.
However, if you wish please feel free to re-open it and provide the clarifications
